srcpkgs/imlib2-heic/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'imlib2-heic'
+pkgname=imlib2-heic
+version=0.1.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="imlib2-devel libheif-devel"
+short_desc="HEIC/HEIF decoder for imlib2 (feh)"
```

Comment:
```suggestion
short_desc="HEIC/HEIF decoder for imlib2 and feh"
```

Maybe? The parentheses aren't obvious...

Reply:
I don't think they meant "and" feh, I think it's meant more like "e.g. feh"

Reply:
See updated description then :)

Reply:
Okay, removed the `(feh)` from `short_desc`